### DISCUSSION TITLE
Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by wforumw** [healthchecks/healthchecks#934] [General]
Hi,
Is it possible to use Nginx Proxy Manager instead of caddy. If yes, is there a place where I get more information about this setup?
Thanks in advance

**Comment by cuu508** [comment]
Yes, I don't see why it would not work. What specific information are you looking for? Have you tried to set it up? What issue(s) did you run into?

**Comment by wforumw** [comment]
Thanks, I took more time to look into, now it works
